.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,25 +4,28 @@ about: Create a report to help us improve
 title: Bug
 labels: bug
 assignees: changchanghwang
```

Comment:
Just curious, Are you going to be in charge of all bug reports, feature requests, and pull requests?

Reply:
yes. for now... if you want, I will add you